packages/block-library/src/separator/theme.scss

```diff
@@ -8,3 +9,8 @@
 		max-width: 100px;
 	}
 }
+
+/* Allows users to understand the clickable area (height) since it is small */
+.wp-block-separator:hover {
+    cursor: pointer;
```

Comment:
Changing the cursor to a hand may make it look like we can drag & drop things around. While when inside a block that's not the case. Not sure if it is ok or we have a better alternative. cc: @jasmussenv in case you have some inputs.

Reply:
Yes, I can understand the thought behind this, but the block should not have separate treatment from any other block.

I think the solution, correctly, is to make the clickable hit area bigger. The fact that we have hover outlines should help.

Reply:
@jasmussen here is another possible solution. I tested this for both `twentyseventeen` and `twentynineteen`. Both show the same result. A few notes in support of adding `!important`

`border-bottom: 2px solid #8f98a1;`, for testing I used a different color which never got recognized without adding `!important`

`border-bottom: 2px solid #8f98a1;` I tried increasing border thickness to `4px` but it never worked without `!important`

It looks to me that some global `hr` styling rules are doing the trick. Once I added `!important`, things started working.

I tried the following modification and it seems to work ok now. Please see the screenshots. @jorgefilipecosta aligning vertically centered seems to be working now as well.

```
.wp-block-separator {
  border: none;
  border-bottom: 2px solid #8f98a1 !important;
  padding: 15px;
  background: none !important; /* had to add this to get rid of the thick gray patch in twenntyseventeen  due to the padding. */
  position: relative;
  bottom: 10px;
  margin: 1.65em auto; }
  .wp-block-separator:not(.is-style-wide):not(.is-style-dots) {
    max-width: 100px; }

```
Screenshot of `twentyseventeen`
![2017](https://user-images.githubusercontent.com/11072678/50154184-995d2080-02ee-11e9-9acd-59509c72b5da.png)

Screenshot of `twentynineteen`
![2019](https://user-images.githubusercontent.com/11072678/50154197-a37f1f00-02ee-11e9-9f58-3e1cd5727610.png)

Reply:
Thank you for the response. It helped me understand why the border is not centered. As you note, it's because it's `border-bottom`. 

I tried a few hacks to center it perfectly and make it more clickable, but found no good solutions. The thing is, a lot of themes built for Gutenberg are now styling the separator assuming it has a border-bottom, so we can't easily change that to, for example, a background color and a height instead. 

For that reason, I don't think we should make a patch for _just this block_.

Instead, we should make a more generic improvement for _all_ blocks. Watch this GIF:

![hover](https://user-images.githubusercontent.com/1204802/50209015-02867780-0373-11e9-8b8b-68650e4df6ef.gif)

Note how the hover outline appears correctly. But in the space between the content and the blue hover outline, you cannot _click_ to select the block. We should make it so you can.

CC: @aduth, I believe you've talked about this in the past too, can you share any insights into how we might go about this?

Reply:
Glad that my observation could help! :) And yes, I understand the limitation. As you said it might be a good idea to create a patch for this block in particular as it might make other themes to suffer!